scripts/migrate_to_deterministic_ids.py

```
@@ -0,0 +1,468 @@
+                vector_service, source_url
+            )
+
+            if not chunks_to_migrate:
+                logger.info(
+                    "No chunks need migration - all IDs are already deterministic!"
+                )
+                return
+
+            # Show preview
+            logger.info(f"Found {len(chunks_to_migrate)} chunks to migrate:")
+            for i, chunk in enumerate(chunks_to_migrate[:5]):  # Show first 5
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

(B007)